Scripts/script-CreateEmailHtmlBody.yml

```diff
@@ -83,7 +95,17 @@ args:
 - name: object
   description: Values object provided as stringified JSON
   isArray: true
+- name: replaceNotFound
+  auto: PREDEFINED
+  predefined:
+  - "yes"
+  - "no"
+  description: if true, replace path not found with empty string, otherwise leave
+    as is
+  defaultValue: "no"
 outputs:
 - contextPath: htmlBody
   description: HTML Body
 scripttarget: 0
+runonce: false
+releaseNotes: added ability to get custom fileds when the path is incident.<customFieldName>, and added an arugment to replace not found values with empty string
```

Comment:
would rephrase to something like:
Added the ability to have custom fields in the template in both .incident.CustomFields.<fieldname> and incident.<fieldname> formats. Also, added the option to replace non-found placeholder values with  empty string